mkdocs.yml

```
@@ -308,8 +308,10 @@ nav:
               - gifit: python_api/plot/gifit.md
               - grid: python_api/plot/grid.md
               - mplstyle: python_api/plot/mplstyle.md
-          - Magic helpers:
-            - hc: python_api/hermitian_conjugate/hc.md
+          - Helpers:
+            - hc: python_api/helpers/hc.md
+            - clicktimes_sse_to_sme: python_api/helpers/clicktimes_sse_to_sme.md
+            - measurements_sse_to_sme: python_api/helpers/measurements_sse_to_sme.md
```

```suggestion
            - clicktimes_to_sme: python_api/helpers/clicktimes_to_sme.md
            - measurements_to_sme: python_api/helpers/measurements_to_sme.md
```
Naming suggestion to make it slightly shorter (arguably less clear as well). Another possibility would be something like `partial_clicktimes` or `incomplete_clicktimes` although that's probably even less clear.

---

Arf, I think the names might be too obscure, although I agree that the current names are quite long. 😕